src/utils/fhir.js

```diff
@@ -447,6 +463,17 @@ function getBaseUrlFromFullUrl(fullUrl) {
   return temp.substring(0, temp.lastIndexOf('/'));
 }
 
+/**
+ * Returns the last part of path from url
+ *  Will return id from url in the form of {baseUrl}/{resourceType}/{id}
```

Comment:
I think this comment is a little misleading. 
```suggestion
 * Returns the last part of path from url
 * For notification endpoints this will be the base64 encoded fullUrl of the PlanDefinition
```

Reply:
Yea I definitely didn't know how to phrase this... I wanted to leave it generic since it can be used for other things.